### DISCUSSION TITLE
SiPM carrier and LM4040 connecttion

### DISCUSSION THREAD

**Discussion by AF1977** [OpenGammaProject/Open-Gamma-Detector#37] [Q&A] [answered]
Hi,

I have two SiPM carrier board v1.0. are the compatible with the detector board v3.x?
Because today I tried to solder an detector board v2.x. Unfortunately I ripped off some copper layer for the LM4040 (VCC conntector).
Now I will buy an fully assembled v3.x from Tindie. Is the PICO included?

Thanks in advance.

BR

Andreas

**Comment by NuclearPhoenixx** [comment, marked as answer]
Hi Andreas,

> I have two SiPM carrier board v1.0. are the compatible with the detector board v3.x? Because today I tried to solder an detector board v2.x.

Yup, they're 100% compatible electrically and functionally. Just watch out for the right pins since the v1 boards come with a 2x3 header and the newer boards only have a couple solder pads.

> Because today I tried to solder an detector board v2.x. Unfortunately I ripped off some copper layer for the LM4040 (VCC conntector).

That's bad, but you don't need to get a new one. Firstly, you don't actually need the voltage reference and secondly, you can solder a cable from the VRef pin of the Pico down to the LM4040 leg and solder the other 2 legs like normal. If you only ripped of the positive pad, it's fine, you can use the board anyways.

> Now I will buy an fully assembled v3.x from Tindie. Is the PICO included?

Yes, of course. The pico is included and already soldered on the board.
